README.md

```diff
@@ -1,4 +1,4 @@
-<img src="/public/img/cvlogo.svg" alt="The CircuitVerse logo" width="736"/> </br></br>
+<img src="/app/assets/images/cvlogo.svg" alt="The CircuitVerse logo" width="736"/> </br></br>
```

Comment:
/app/assets cannot be hardcoded

Reply:
This is readme, so not an issue. But hardcoding app/assets inside the app won't work

Reply:
exactly, since README is outside the app, I needed to hard code that.